lib/cisco_node_utils/bgp_neighbor.rb

```diff
@@ -431,7 +432,7 @@ def default_transport_passive_mode
 
     def transport_passive_only=(val)
       # TODO: add "deprecated" warning?
-      self.transport_passive_mode = (val ? :passive_only : :both)
+      self.transport_passive_mode = (val ? :passive_only : :none)
```

Comment:
The getters wrap this in a `:nexus` check - is the setter safe for both platforms?